src/pages/learn/[id].tsx

```diff
@@ -96,7 +93,7 @@ function LearnDetail() {
   const { setOrder, setText, setClickedDeleteType, nodeToText } = useDeleteElement({
     rightClickedElement,
     clickedTitleIndex,
-    detailId,
+    detailId: String(detailId),
```

Comment:
```suggestion
    detailId: Number(detailId),
```

Number로 바꿔주는 것은 어떨까요?
useDeleteElement 쪽 코드를 다시 보니 getPrivateVideoData에 detailId가 사용되고 있는데
string으로 전달했다가 useDeleteElement에서 다시 Number로 감싸는 것보다
처음부터 Number로 보내면 좋을 것 같아서 제안드립니다!